testing/py3-empy/APKBUILD

```diff
@@ -0,0 +1,24 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname=py3-empy
+_pkgname=empy
+pkgver=3.3.4
+pkgrel=0
+pkgdesc="A powerful and robust templating system for Python."
+url="http://www.alcyone.com/software/empy/"
+arch="noarch"
+options="!check" # no test suite
+license="LGPL-3.0-or-later"
+depends="python3"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
```

Comment:
http://www.alcyone.com/software/empy/empy-$pkgver can be used instead which contains a test.sh file (i didn't test if it works) and also reveals that the license is LGPL-2.1-or-later

Reply:
Good catch. I had a quick look at it. Seems really crude and, if I am not mistaken, will need patching to use `python3` and to `exit 1` if the script fails. Should I create a patch for it?